site/InteractionNotice.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+
+// A modified FontAwesome icon. This is a string to be used outside of React
```

Comment:
Can you explain what you mean by WordPress content here? Won't we be using this in gdocs baking too?

Reply:
Reworded. Is it better now?